### ISSUE TITLE
stuck in black

### ISSUE THREAD

**Issue by smoky6969** [BetterDiscord/Installer#310]
when i open the aps stuck in black
![image](https://user-images.githubusercontent.com/120582662/207637073-b6386ffb-57a5-47eb-931e-29cf131ef4bc.png)

**Comment by tnbhaeufi** [comment]
got the same problem. never had that before. Using Radeon RDNA2 graphics card

**Comment by aaxim** [comment]
even if you unblock it in properties to bypass with windows virus thing it still does it.

**Comment by The-Mighty-Spoon** [comment]
Having the same issue

**Comment by Dethkiller15** [comment]
While I had the same issue I did just give up and use the manual install method. I just followed along with the video.
https://cdn.discordapp.com/attachments/1019725965284360283/1042235899246817332/Manualinstall-handbrake.mp4

Here is a link to the page.
https://docs.betterdiscord.app/users/getting-started/installation/#manual-installation

**Comment by terax6669** [comment]
@Tropix126 @rauenzi @Inve1951
![image](https://user-images.githubusercontent.com/19204073/208407660-ec49b3eb-7ef1-45c4-95ea-9eae2837102e.png)

**Comment by neobenedict** [comment]
Same issue.

**Comment by RaySollium99** [comment]
Same issue over here

**Comment by Deceron** [comment]
same issue still happening

**Comment by noassl** [comment]
Which node version are you using? Switching to v17 solved this issue for me.

**Comment by terax6669** [comment]
> Which node version are you using? Switching to v17 solved this issue for me.

I don't see how your development environment could impact the release binary, but... I've tried downgrading from 18 to 16 and it did nothing.

**Comment by Johnnycyan** [comment]
I have the same issue.

OS: Windows 11 Pro 22H2 Build 22621.1105
GPU: Nvidia RTX 3090
CPU: Ryzen 5950x
RAM: 64GB CL18 3600Mhz

**Comment by Dethkiller15** [comment]
> I have the same issue.
> 
> OS: Windows 11 Pro 22H2 Build 22621.1105 GPU: Nvidia RTX 3090 CPU: Ryzen 5950x RAM: 64GB CL18 3600Mhz

Best bet is to go to your betterdiscord folder and delete everything in it but the plugins and themes folders.
then uninstall discord itself and make sure it is 100% uninstalled by making sure the discord folder isnt in appdata. Then reinstall discord and then betterdiscord. Should work. I did the same thing on my system. If it works for you then please tell us.

**Comment by xxtraspicyy** [comment]
have attempted the manual install as mentioned in this thread and have also deleted all appdata folders for betterdiscord and discord etc still same issue

**Comment by chrisrtxv** [comment]
This is still occurring on the installer for 1.8.5.

**Comment by zerebos** [comment]
Duplicate of #302
